### DISCUSSION TITLE
Add filter options to listview

### DISCUSSION THREAD

**Discussion by bjarnef** [umbraco/Umbraco-CMS#11362] [Features and ideas]
When having more than a single content type in a listview, it would be useful to be able to filter these similar the users overview.

Mainly filter on the content type, but it may also make sense to filter on other data, e.g. published/unpublished (only relevant to content nodes).

Furthermore it would be cool if the listview was configurable, so you with a simple function could build custom filters, e.g. "Show only news articles with a release date (date picker)" or "Show employees with photo (media picker 3)".

**Comment by patrickdemooij9** [comment]
Think this kinda comes back to the idea by @nzdev right? https://github.com/umbraco/Umbraco-CMS/discussions/11302

**Reply by bjarnef** [reply]
Yeah I saw that and thought it was related. Maybe we get some more advanced/complex search functionality later, but I think this depends on the specific properties the content types have, which not neccesary are the same for the different content types shown in the listview. 

What we do know is that content types always have an alias, so it could be a start to be able to filter on content types. Maybe enabled if more than one content types are allowed under a patent node, but could be configurable.

**Reply by nzdev** [reply]
My plan would be to implement the headless option from https://github.com/elastic/search-ui.
